pkg/operator/ceph/agent/agent.go

```diff
@@ -231,6 +232,7 @@ func (a *Agent) createAgentDaemonSet(namespace, agentImage, serviceAccount strin
 	// Add toleration if any
 	tolerationValue := os.Getenv(agentDaemonsetTolerationEnv)
 	if tolerationValue != "" {
+		logger.Warningf("%s and %s are deprecated. Use %s instead.", agentDaemonsetTolerationEnv, agentDaemonsetTolerationKeyEnv, agentDaemonsetTolerationsEnv)
```

Comment:
We can't deprecate functionality that might affect users during upgrade. I think the best thing to do is to support the old single-toleration env var as well as the new one introduced here.

Reply:
Thanks, fixed.